conftest.py

```diff
@@ -1,28 +1,45 @@
 """Setup for Pytest."""
 
 # Standard library imports
+from pathlib import Path
+import shutil
+from sys import platform
 import warnings
 import webbrowser
```

Comment:
```suggestion
import shutil
import sys
import warnings
import webbrowser
from pathlib import Path
```

`from` below `import` is the most standard style AFAIK and the `isort` default, and what we use in the other modules.